### PR TITLE
Add Longest Word in R

### LINKED ISSUE #4021: Add Longest Word in R
To request a new code snippet, please fill out the following:

Project name: Longest Word
Project link: https://sampleprograms.io/projects/longest-word/
Language: R

### PR DESCRIPTION
**Pull request by TavSingh**
Congrats on taking the first step to contributing to the Sample Programs repository maintained by [The Renegade Coder][renegade-coder]! 
For simplicity, please make sure that your pull request includes one and only one contribution.

Please fill _one_ of the sections below as applicable.
Please also add any other relevant information to the Notes section at the bottom.
You may delete or just ignore any other sections.
For more information please refer to our [contributing documentation][contributing]

## I Am Adding a New Code Snippet in an Existing Language

- [x] I fixed #4021
- [x] I named the pull request using `Add {PROJECT} in {LANGUAGE}` format
  
  

[renegade-coder]: https://therenegadecoder.com/
[contributing-plagiarism]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#plagiarism
[contributing-new-project]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#requirements-for-a-new-project
[contributing-new-language]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#requirements-for-a-new-language
[contributing-readme]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#create-readmes
[contributing-tests-in-detail]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#tests-in-detail
[contributing]: CONTRIBUTING.md
[sample-programs-project-list]: https://sampleprograms.io/projects/
[contributing-modifications]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#modifying-existing-code-snippets

### CODE REVIEW

**Review by rzuckerm — CHANGES_REQUESTED** [review]
Please remove the extra files. The only file that should be in the pull request is `longest-word.R`

**Comment by TavSingh** [comment]
Extra files have been removed

File: archive/r/r/longest-word.R [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  
+  if (length(args) == 0 || args == "") {
+    cat("Usage: please provide a string\n")
+  } else {
+    # Split the string by whitespace and find the maximum word length in one line
+    cat(max(nchar(unlist(strsplit(args[1], "[ \t\n\r]+")))), "\n")
+  }
+}
+
+longest_word_length()
```
Comment (rzuckerm) [inline archive/r/r/longest-word.R:13]: This needs to accept a command-line argument. Please see the [project documentation](https://sampleprograms.io/projects/longest-word/)

**Review by rzuckerm — CHANGES_REQUESTED** [review]

**Comment by TavSingh** [comment]
Accepts command line noq

**Review by rzuckerm — APPROVED** [review]
Looks good, @TavSingh . Thanks for your contribution!

**Pull request merged**